servers/visual/rasterizer.h

```diff
@@ -1256,6 +1272,34 @@ class RasterizerCanvas {
 class Rasterizer {
 protected:
 	static Rasterizer *(*_create_func)();
+	static bool khr_debug_enabled;
+
+	//Profiling Information
+	//Labels
+	static void (*push_label_func)(unsigned int source, unsigned int id, int length, const char *message);
+	static void (*pop_label_func)();
+
+	//Timing stuff
+#define FRAME_TIMING_COUNT 3
+#define MAX_QUERIES 128
+#define GL_QUERY_RESULT 0x8866
+#define GL_TIMESTAMP 0x8E28
+
```

Comment:
Can we use enums / consts instead of defines here? Defines is usually bad practice for constants, especially in headers.